cmd/api.go

```diff
@@ -48,6 +48,9 @@ func (o *apiOptions) bindCmdFlags(cmd *cobra.Command) {
 	cmd.Flags().BoolVar(&o.apiScaffolder.DoController, "controller", true,
 		"if set, generate the controller without prompting the user")
 	o.controllerFlag = cmd.Flag("controller")
+	cmd.Flags().BoolVar(&o.apiScaffolder.DoWebhook, "webhook", false,
```

Comment:
do we want to differentiate from "manual" webhooks (i.e. not our special defaulting/validating)?

Reply:
Discussed with Solly offline. We can have something like `kubebuilder create webhook --low-level-defaulting --low-level-programmatic-validation` for "manual" webhooks if we want to do it in the future.

Reply:
Filed tracking issue https://github.com/kubernetes-sigs/kubebuilder/issues/833